packaging/irods.service.template

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=iRODS 5
+After=network.target
+
+[Service]
+Type=forking
+PIDFile=/run/irods/irods-server.pid
+ExecStart=/usr/sbin/irodsServer -d
+ExecReload=/bin/kill -HUP $MAINPID
+ExecStop=/bin/kill -TERM $MAINPID
+Restart=on-failure
+User=irods
+Group=irods
+WorkingDirectory=/var/lib/irods
+LimitNOFILE=65536
```

Comment:
ping for @SwooshyCueb 

do we have an opinion about this?

Reply:
Bumping that value is acceptable/good and aligns with other applications.

Bumping.

Reply:
Bumped to 1048576.